lib/render/editor/editor.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
```

Comment:
This file will be included in the final laboratory's output, so it must have only the `<link>` and `<dom-module>` tags, and not the others (`<!doctype>, <html>, <head>, <body>, ...`)